practice/target/maven-archiver/pom.properties

```diff
@@ -0,0 +1,5 @@
+#Generated by Maven
+#Mon Mar 16 20:52:35 UTC 2020
+version=1.0-SNAPSHOT
+groupId=ca.jrvs.apps
+artifactId=practice
```

Comment:
please do not commit generated files such as ` practice/target/practice-1.0-SNAPSHOT.jar`

Reply:
please run `git rm remove practice/target`

Reply:
BTW. please click resolve conversation when you fixed a comment